arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -1325,5 +1325,133 @@ public async Task PutEdgeDefinitionAsync_ShouldThrow_WhenEdgeCollectionNameDoesN
             Assert.Equal(HttpStatusCode.NotFound, ex.ApiError.Code);
             Assert.Equal(1930, ex.ApiError.ErrorNum); // GRAPH_EDGE_COLLECTION_NOT_USED
         }
+
+        [Fact]
+        public async Task PutVertexAsync_ShouldSuceed()
+        {
+            string graphName = nameof(PutVertexAsync_ShouldSuceed) + "_graph";
+
+            PostGraphResponse createGraphResponse = await _client.PostGraphAsync(
+                new PostGraphBody()
+                {
+                    Name = graphName
+                });
+
+            Assert.Equal(HttpStatusCode.Accepted, createGraphResponse.Code);
+
+            string vertexClx = nameof(PutVertexAsync_ShouldSuceed) + "_clx";
+
+            PostVertexCollectionResponse createVertexClxResponse = await _client.PostVertexCollectionAsync(
+                graphName,
+                new PostVertexCollectionBody()
+                {
+                    Collection = vertexClx
+                });
+
+            var beforeVertexClxVtx = vertexClx + "_vtx";
+            var afterVertexClxVtx = vertexClx + "_vtx_2";
+
+            var createVertexResponse = await _client.PostVertexAsync(graphName, vertexClx, new
+            {
+                Name = beforeVertexClxVtx
+            });
+            var response = await _client.PutVertexAsync(graphName, vertexClx, createVertexResponse.Vertex._key, new PutVertexMockModel
+            {
+                Name = afterVertexClxVtx
+            }, new PutVertexQuery
+            {
+                ReturnNew = true,
+                ReturnOld = true,
+                WaitForSync = true
+            });
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.False(response.Error);
+            Assert.Equal(afterVertexClxVtx, response.New.Name);
+            Assert.NotEqual(response.New.Name, response.Old.Name);
+            Assert.NotEqual(createVertexResponse.Vertex._rev, response.Vertex._rev);
+        }
+
+        [Fact]
+        public async Task PutVertexAsync_ShouldThrow_WhenGraphIsNotFound()
+        {
+            string graphName = nameof(PutVertexAsync_ShouldThrow_WhenGraphIsNotFound);
+            string vertexClx = nameof(PutVertexAsync_ShouldThrow_WhenGraphIsNotFound);
+            string fakeGraph = graphName + "_bogus";
+            PostGraphResponse createGraphResponse = await _client.PostGraphAsync(
```

Comment:
I don't think you need to create any graph or vertex before doing `_client.PutVertexAsync("bogusGraph"...`.